lib/python/picongpu/picmi/gaussian_laser.py

```diff
@@ -42,31 +84,56 @@ def get_as_pypicongpu(self) -> laser.GaussianLaser:
         # unsupported: fill_in (do not warn, b/c we don't know if it has been
         # set explicitly, and always warning is bad)
 
-        assert self.centroid_position[0] == self.focal_position[0] \
-            and self.centroid_position[2] == self.focal_position[2], \
-            "focal position x and z MUST be equal to centroid x and z"
-        assert [0, 1, 0] == self.propagation_direction, \
-            "only support propagation along Y axis"
-        assert 0 == self.centroid_position[1], "centroid MUST have y=0"
+        assert abs(self.scalarProduct(self.polarization_direction,
+                self.polarization_direction) - 1) < 1e-9, \
+            "the polarization direction vector must be normalized"
+
+        # check for excessive phase values to avoid numerical precision errors
+        assert abs(self.picongpu_phase) <= 2*3.1416, \
```

Comment:
Do we need a hard coded PI, I assume python has a variable for that.

Reply:
Done